veracruz-utils/src/csr.rs

```diff
@@ -153,7 +153,7 @@ pub const COMPUTE_ENCLAVE_CSR_TEMPLATE: CsrTemplate = CsrTemplate {
     signature_location: (237, 237 + 71),
     signature_range: (4, 222),
     overall_length_field_location: (2, 4),
-    overall_length_initial_value: 233,
+    overall_length_initial_value: 237 - 4,
```

Comment:
What is this calculation?

Reply:
These are all over the `csr.rs` due to how we are templating the CSRs and certificates. With the removal of the root enclaves, much of that templating will go away. `237-4` is the overall length of the template, minus the 4 bytes for the tag, length encoding at the beginning. There are many examples of that in this file, and... it's all messy.

After the root enclave removal, transitioning to using MbedTLS inside the enclaves will result in the final removal of this mess. All of this really roots from our use of `rustls` and that it doesn't support certificate management in any way.